testing/pkg_test_demo/test/example_test.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:test/test.dart';
```

Comment:
nit: Needs the engine copyright header.

(Hopefully when we finish the buildmoot, this can be enforced by a tool.)